### PR TITLE
Update electron mass

### PR DESCRIPTION
**Pull request by timbrisc**

### CODE REVIEW

**Comment by timbrisc** [comment]
#231

**Review by swagers — APPROVED** [review]

**Comment by swagers** [comment]
@timbrisc, this has conflict can you resolve it?

File: ucum-source.xml [hunk 1]
```diff
@@ -1707,7 +1707,7 @@ out of scope of &TUCUM;.
...
 <u:unit id="300023" Code="[mu_0]" CODE="[MU_0]" isMetric="yes"><name>permeability of vacuum</name><printSymbol><i>&mu;<sub><r>0</r></sub></i></printSymbol><property>magnetic permeability</property><value Unit="4.[pi].10*-7.N/A2" UNIT="4.[PI].10*-7.N/A2" value="1">1</value></u:unit>
 <u:unit id="200190" Code="[e]" CODE="[E]" isMetric="yes"><name>elementary charge</name><printSymbol><i>e</i></printSymbol><property>electric charge</property><value Unit="C" UNIT="C" value="1.60217733e-19">1.60217733 &mult; 10<sup>-19</sup></value></u:unit>
-<u:unit id="300022" Code="[m_e]" CODE="[M_E]" isMetric="yes"><name>electron mass</name><printSymbol><i>m<sub><r>e</r></sub></i></printSymbol><property>mass</property><value Unit="g" UNIT="g" value="9.1093897e-28">9.1093897 &mult; 10<sup>-28</sup></value></u:unit>
+<u:unit id="300022" Code="[m_e]" CODE="[M_E]" isMetric="yes"><name>electron mass</name><printSymbol><i>m<sub><r>e</r></sub></i></printSymbol><property>mass</property><value Unit="g" UNIT="g" value="9.1093837015-28">9.1093837015 &mult; 10<sup>-28</sup></value></u:unit>
```
Comment (swagers) [inline ucum-source.xml]: Should the e be removed from the value?

**Review by swagers — CHANGES_REQUESTED** [review]

File: ucum-source.xml (hunk 1 above)
Comment (timbrisc) [reply ucum-source.xml]: You are correct. It needs the e. I'll address.

**Comment by timbrisc** [comment]
The exponent was wrong as well. Here is the updated value for reference: https://physics.nist.gov/cgi-bin/cuu/Value?me

**Review by swagers — APPROVED** [review]

**Review by chgessner — COMMENTED** [review]
For electron mass the reference unit used by NIST is kg - not g. This shall be corrected, or the exponent of the numerical value changed appropriately. (value Unit="kg" UNIT="kg" )

**Comment by timbrisc** [comment]
@chgessner Thank you for identifying this error. The change has been reflected in the release/2.2 branch (release candidate).

**Pull request merged**